sdk/tables/aztable/table_client.go

```diff
@@ -58,14 +58,35 @@ func (t *TableClient) Delete(ctx context.Context) (TableDeleteResponse, error) {
 // pager := client.Query(nil)
 // for pager.NextPage(ctx) {
 //     resp = pager.PageResponse()
-//     fmt.sprintf("The page contains %i results", len(resp.TableEntityQueryResponse.Value))
+//     fmt.Fprintf("The page contains %i results", len(resp.TableEntityQueryResponse.Value))
 // }
 // err := pager.Err()
 func (t *TableClient) Query(queryOptions *QueryOptions) TableEntityQueryResponsePager {
 	if queryOptions == nil {
 		queryOptions = &QueryOptions{}
 	}
-	return &tableEntityQueryResponsePager{tableClient: t, queryOptions: queryOptions, tableQueryOptions: &TableQueryEntitiesOptions{}}
+	return &tableEntityQueryResponsePager{
+		tableClient:       t,
+		queryOptions:      queryOptions,
+		tableQueryOptions: &TableQueryEntitiesOptions{}}
+}
+
+// List returns all the entities in a table
+//
+// List returns a Pager, which allows iteration through each page of results. Example:
+//
+// pager := client.List()
+// for pager.NextPage(ctx) {
+//     resp = pager.PageResponse()
+//     fmt.Fprintf("The page contains %i results", len(resp.TableEntityQueryResponse.Value))
+// }
+// err := pager.Err()
+func (t *TableClient) List() TableEntityQueryResponsePager {
```

Comment:
What's the difference between `List` and `Query`?

Reply:
List is just an empty query. In Python we have both even though both can be done with just `Query`. I didn't see anything in the Go guidelines about not including a List so I have it here. I'll talk to Jeffrey about what our language guideline is.

Reply:
Ok cool. BTW, if we leave it, should the implementation just be:
```go
{
 return t.Query();
```